Devsite proposta Q2-CORE (#756)
mercadopago/devsite-docs: guides/plugins/unofficial/core.pt.md

---
 sites_supported:
  - mlb
---

# Linx Commerce

## O que é Linx Commerce?

É uma **plataforma virtual que te permite receber pagamentos com o Mercado Pago**.

> Com ela você poderá, oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_crédito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) no próprio checkout da sua loja.

Reviewer: ````suggestion
Com ela você poderá, oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_crédito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) na sua loja.
````

## Etapas para configurar

Os passos para **começar a receber com Mercado Pago**, são os seguintes:

1. Crie uma [conta de vendedor](https://www.mercadopago[FAKER][URL][DOMAIN]/activities) no Mercado Pago caso ainda não tenha uma.

> 2. Adicione o Mercado Pago como meio de pagamento dentro da sua loja.

Reviewer: ````suggestion
1. Adicione o Mercado Pago como meio de pagamento dentro da sua loja.
````

> 3. Configure a forma de pagamento como Mercado Pago.

Reviewer: ````suggestion
1. Configure a forma de pagamento como Mercado Pago.
````

> 4. Configure o meio de pagamento do contrato.

Reviewer: ````suggestion
1. Configure o meio de pagamento do contrato.
````

## Adicione o Mercado Pago como meio de pagamento

Para **adicionar o Mercado Pago em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.

> 2. Acesse “Adicionar meio de pagamento". 

Reviewer: ````suggestion
1. Acesse “Adicionar meio de pagamento". 
````

> 3. Na lista Fornecedor de Serviço, selecione o "Mercado Pago V2" e clique em "Próximo Passo".

Reviewer: ````suggestion
1. Na lista Fornecedor de Serviço, busque por Mercado Pago e clique em "Próximo Passo".
````

> 4. Defina um nome para o meio de pagamento Mercado Pago V2 e selecione o status "Ativo".

Reviewer: ````suggestion
1. Defina um nome para o meio de pagamento Mercado Pago V2 e selecione o status "Ativo".
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_adicione_meio_pagamento-1.gif)
<p>&nbsp;</p>

## Configure a forma de pagamento

Depois de adicionar o Mercado Pago, você terá a opção de oferecer pagamentos com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_crédito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto).

Além disso, você pode [oferecer parcelas sem juros](#bookmark_configurando_o_parcelamento_na_sua_conta_do_mercado_pago), consultar e configurar as [taxas e prazos](https://www.mercadopago[FAKER][URL][DOMAIN]/settings/release-options/) de suas vendas online quando quiser.

### Configurando os dados da integração

Após adicionar o meio de pagamento, **configure os dados de sua integração com o Mercado Pago** através dos seguintes passos:

1. Acesse a aba "Integração" e complete os campos Public Key e Access Token com as [credenciais de produção]([FAKER][CREDENTIALS][URL]) de sua conta do Mercado Pago.

> 2. Preencha o nome que será informado na fatura do seu cliente para que ele possa reconhecê-lo.

Reviewer: ````suggestion
1. Preencha o nome que será informado na fatura do seu cliente para que ele possa reconhecê-lo.
````

> 3. Se deseja habilitar pagamentos de teste, na seção **Modo teste**, selecione a opção "sim". Depois complete os campos Public Key e Access Token com as [credenciais de teste]([FAKER][CREDENTIALS][URL]) da sua conta do Mercado Pago.

Reviewer: ````suggestion
1. Se deseja habilitar pagamentos de teste, na seção **Modo teste**, selecione a opção "Sim". Depois complete os campos Public Key e Access Token com as [credenciais de teste]([FAKER][CREDENTIALS][URL]) da sua conta do Mercado Pago.
````

> 4. Na seção de Parcelamento, selecione "Externo (API do Mercado Pago)" onde você vai utilizar as condições de parcelamento diretamente da sua conta Mercado Pago.

Reviewer: ````suggestion
1. Na seção de Parcelamento, selecione "Externo (API do Mercado Pago)" onde você vai utilizar as condições de parcelamento diretamente da sua conta Mercado Pago.
````

> 5. Clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_integracao-2.gif)
<p>&nbsp;</p>

> Encontre todas as informações sobre suas credenciais em [perguntas frequentes](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/faqs/credentials/).

### Configurando os dados de cartão de crédito

Para **configurar os dados de Cartão de Crédito em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.

> 2. Localize o meio de pagamento criado e clique em "Editar".

Reviewer: ````suggestion
1. Localize o meio de pagamento criado e clique em "Editar".
````

> 3. Acesse a aba "Cartões de Crédito", escolha a bandeira que deseja configurar e selecione o status "Ativo".

Reviewer: ````suggestion
1. Acesse a aba "Cartões de Crédito", escolha a bandeira que deseja configurar e selecione o status "Ativo".
````

> 4. Preencha os campos correspondentes, se necessário.

Reviewer: ````suggestion
1. Preencha os campos correspondentes, se necessário.
````

    | Campos | Dados |
    | --- | --- |
    | Porcentagem de juros | Caso queira adicionar uma porcentagem de juros, preencha um valor, caso não, deixe em 0,00. |
    | Tipo de juros | Emissor - Incluir juros no total do pedido. Emissor - Juros acrescido pelo emissor. Lojista - Incluir juros no total do pedido. |
    | Número total de parcelas | Coloque o número total de parcelas que será possível escolher por compra. |
    | Número de parcelas sem juros | O número total de parcelas sem juros que será possível escolher por compra. |
    | Parcela mínima | Valor mínimo disponível para cada parcela de uma compra. |
    | Identificador de Integração | Coloque um código que será identificado no seu ERP. |
    | Ordem | Configure a ordem em que as bandeiras aparecerão no Checkout. |
    | Caminho Imagem | Complete o caminho onde as imagem das bandeiras do cartão serão buscadas. |

> 5. Clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

> WARNING
>
> Importante
>
> Ao utilizar configuração de parcelamento externo, todas as configurações de parcelamento realizadas na plataforma serão desconsideradas.<br>
> O parcelamento externo não será mostrado em listagens ou no detalhe do produto.

> 

Reviewer: ````suggestion
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_cartao-3.gif)
<p>&nbsp;</p>

### Configurando os dados de Boleto

Para **configurar os dados de Boleto Bancário o em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.

> 2. Localize o meio de pagamento criado e clique em "Editar".

Reviewer: ````suggestion
1. Localize o meio de pagamento criado e clique em "Editar".
````

> 3. Acesse a aba "Boletos" e selecione o status para "Ativo".

Reviewer: ````suggestion
1. Acesse a aba "Boletos" e selecione o status para "Ativo".
````

> 4. Preencha o campo **Identificador de Integração** com a informação que será identificada em seu ERP.

Reviewer: ````suggestion
1. Preencha o campo **Identificador de Integração** com a informação que será identificada em seu ERP.
````

> 5. Informe a ordem em que o boleto aparecerá no seu checkout.

Reviewer: ````suggestion
1. Informe a ordem em que o boleto aparecerá no seu checkout.
````

> 6. Por último, clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_boleto-4.gif)
<p>&nbsp;</p>

## Configurando o parcelamento na sua conta do Mercado Pago

1. Na seção [Seu negócio](https://www.mercadopago.com.br/business/) na sua conta do Mercado Pago.

> 2. Procure por **Oferecer parcelamento sem juros** e clique em "Ativar".

Reviewer: ````suggestion
1. Procure por **Oferecer parcelamento sem juros** e clique em "Ativar".
````

> 3. Escolha quantas parcelas você quer oferecer e clique em “Ativar” para confirmar as alterações.

Reviewer: ````suggestion
1. Escolha quantas parcelas você quer oferecer e clique em “Ativar” para confirmar as alterações.
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_parcelamento_conta-5.gif)
<p>&nbsp;</p>

E pronto! Você está oferecendo parcelamento sem juros, assumindo as tarifas de parcelamento que você configurou.

## Configurando meios de pagamento do contrato

O contrato tem a finalidade de determinar configurações específicas que serão utilizadas pela sua loja.

### Criando um contrato

Para **criar um contrato em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Contratos" no menu Backoffice.

> 2. Clique em “Adicionar Contrato".

Reviewer: ````suggestion
1. Clique em “Adicionar Contrato".
````

> 3. Preencha um nome para o contrato que está sendo criado.

Reviewer: ````suggestion
1. Preencha um nome para o contrato que está sendo criado.
````

> 4. Por último, clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_criando_um_contrato-6.gif)
<p>&nbsp;</p>

### Definindo os meios de pagamento do contrato

Para **definir os meios de pagamento do contrato**, siga os passos abaixo:

1. No painel administrativo, acesse "Contratos" no menu Backoffice.

> 2. Localize o contrato criado e clique em "Editar".

Reviewer: ````suggestion
1. Localize o contrato criado e clique em "Editar".
````

> 3. Localize a sessão meios de pagamento e defina a opção de contrato que deseja utilizar:

Reviewer: ````suggestion
1. Localize a sessão meios de pagamento e defina a opção de contrato que deseja utilizar:
````

> 3. Localize a sessão meios de pagamento e defina qual a opção que o contrato quer refletir:

Reviewer: ````suggestion
3. Localize a sessão meios de pagamento e defina qual a opção que o contrato quer utilizar:
````

> 3. Localize a sessão meios de pagamento e defina qual a opção que o contrato quer utilizar:

Reviewer: ````suggestion
3. Localize a sessão meios de pagamento e defina a opção de contrato que deseja utilizar:
````

> 

Reviewer: ````suggestion
````

> - **Todos os meios de pagamento ativos na plataforma**. Irá permitir que todos os meios de pagamento habilitados na plataforma.

Reviewer: ````suggestion
    - **Todos os meios de pagamento ativos na plataforma**. Irá permitir que todos os meios de pagamento habilitados na plataforma.
````

>     - **Todos os meios de pagamento ativos na plataforma**. Irá permitir que todos os meios de pagamento habilitados na plataforma.

Reviewer: ````suggestion
- **Todos os meios de pagamento ativos na plataforma**. Irá permitir que todos os meios de pagamento habilitados na plataforma.
````

> - **Definidos abaixo**. Escolher quais meios de pagamento irão fazer parte do contrato.

Reviewer: ````suggestion
    - **Definidos abaixo**. Escolher quais meios de pagamento irão fazer parte do contrato.
````

> 

Reviewer: ````suggestion
````

>     - **Definidos abaixo**. Escolher quais meios de pagamento irão fazer parte do contrato.

Reviewer: ````suggestion
- **Definidos abaixo**. Escolher quais meios de pagamento irão fazer parte do contrato.
````

> 4. Clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_meio_pagamento_contrato-7.gif)
<p>&nbsp;</p>

### Atribuindo meio de pagamento do contrato a um canal

Para **atribuir o meio de pagamento do contrato ao canal de sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Canais" no menu Canais.

> 2. Localize o canal da sua loja e clique em "Editar".

Reviewer: ````suggestion
1. Localize o canal da sua loja e clique em "Editar".
````

> 3. Na sessão contrato padrão, adicione o contrato que criou.

Reviewer: ````suggestion
1. Na sessão contrato padrão, adicione o contrato que criou.
````

> 4. Clique em "Salvar".

Reviewer: ````suggestion
4. Por último, clique em "Salvar".
````

> 4. Por último, clique em "Salvar".

Reviewer: ````suggestion
1. Por último, clique em "Salvar".
````

> 

Reviewer: ````suggestion
````

1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.
2. Localize o meio de pagamento criado e clique em "editar".
3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar, e selecione o status "Ativo".
4. Preencha os campos correspondentes, se necessário, e clique em "Salvar".

| Campos | Dados necessários |
|-------------------------------------------|---------------------------------------------------------------------------------------|
| Porcentagem de juros | Caso queira adicionar uma porcentagem de juros, preencha um valor, caso não, deixe em 0,00. |
| Tipo de juros | Emissor - Incluir juros no total do pedido. Emissor - Juros acrescido pelo emissor. Lojista - Incluir juros no total do pedido. |
| Número total de parcelas | Coloque o número total de parcelas que será possível escolher por compra. |
| Número de parcelas sem juros | O número total de parcelas sem juros que será possível escolher por compra. |
| Parcela mínima | Valor mínimo disponível para cada parcela de uma compra. |
| Identificador de Integração | Coloque um código que será identificado no seu ERP. |

> | Ordem | A ordem em que as bandeiras aparecerão no Checkout. |

Reviewer: ````suggestion
    | Ordem | A ordem em que as bandeiras aparecerão no Checkout. |
````

> | Caminho Imagem | O caminho onde as imagem das bandeiras do cartão serão buscadas. |

Reviewer: ````suggestion
    | Caminho Imagem | Complete o caminho onde as imagem das bandeiras do cartão serão buscadas. |
````

> <br>

Reviewer: ````suggestion
````

> 

Reviewer: ````suggestion
5. Clique em "Salvar".
````

> WARNING
>
> Importante
>

> > Ao utilizar configuração de parcelamento externo, todas as configurações de parcelamento realizadas na plataforma serão desconsideradas.

Reviewer: ````suggestion
> Ao utilizar configuração de parcelamento externo, todas as configurações de parcelamento realizadas na plataforma serão desconsideradas.<br>
````

> > O parcelamento externo **não** será mostrado em listagens ou no detalhe do produto.

Reviewer: ````suggestion
> O parcelamento externo não será mostrado em listagens ou no detalhe do produto.
````

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_cartao-3.gif)
<p>&nbsp;</p>

### Configurando os dados de Boleto

Para **configurar os dados de Boleto Bancário o em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.

> 2. Localize o meio de pagamento criado e clique em "editar".

Reviewer: ````suggestion
2. Localize o meio de pagamento criado e clique em "Editar".
````

3. Acesse a aba "Boletos" e selecione o status para "Ativo".
4. Preencha o campo **Identificador de Integração** com a informação que será identificada em seu ERP.

> 5. Informe a ordem em que o boleto aparecerá no seu checkout e clique em "Salvar".

Reviewer: ````suggestion
5. Informe a ordem em que o boleto aparecerá no seu checkout.
6. Por último, clique em "Salvar".
````

Reviewer: se esse der conflito, adicionamos o item 6 manualmente no md

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_boleto-4.gif)
<p>&nbsp;</p>

## Configurando o parcelamento na sua conta do Mercado Pago

> 1. Acesse a seção [Seu negócio](https://www.mercadopago.com.br/business/) na sua conta do Mercado Pago.

Reviewer: ````suggestion
1. Na seção [Seu negócio](https://www.mercadopago.com.br/business/) na sua conta do Mercado Pago.
````

2. Procure por **Oferecer parcelamento sem juros** e clique em "Ativar".
3. Escolha quantas parcelas você quer oferecer e clique em “Ativar” para confirmar as alterações.
<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_parcelamento_conta-5.gif)
<p>&nbsp;</p>

E pronto! Você está oferecendo parcelamento sem juros, assumindo as tarifas de parcelamento que você configurou.

## Configurando meios de pagamento do contrato

O contrato tem a finalidade de determinar configurações específicas que serão utilizadas pela sua loja.

### Criando um contrato

Para **criar um contrato em sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Contratos" no menu Backoffice.
2. Clique em “Adicionar Contrato".

> 3. Preencha um nome para o contrato que está sendo criado e clique em "salvar".

Reviewer: ````suggestion
3. Preencha um nome para o contrato que está sendo criado.
4. Por último, clique em "Salvar".
````

````

Reviewer: se esse der conflito, adicionamos o item 4 manualmente no md

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_criando_um_contrato-6.gif)
<p>&nbsp;</p>

### Definindo os meios de pagamento do contrato

> Para **definir os meios de pagamento do contrato em sua loja**, siga os passos abaixo:

Reviewer: ````suggestion
Para **definir os meios de pagamento do contrato**, siga os passos abaixo:
````

1. No painel administrativo, acesse "Contratos" no menu Backoffice.

> 2. Localize o contrato criado e clique em "editar".

Reviewer: ````suggestion
2. Localize o contrato criado e clique em "Editar".
````

> 3. Localize a sessão meios de pagamento e defina qual a opção que o contrato irá refletir para o meio de pagamento, sendo eles:

Reviewer: ````suggestion
3. Localize a sessão meios de pagamento e defina qual a opção que o contrato quer refletir:
````

> - **Todos os meios de pagamento ativos na plataforma.** Irá permitir que todos os meios de pagamento habilitados na plataforma sejam refletidos para este contrato.

Reviewer: ````suggestion
- **Todos os meios de pagamento ativos na plataforma**. Irá permitir que todos os meios de pagamento habilitados na plataforma.
````

> - **Definidos abaixo.** Escolher quais meios de pagamento irão fazer parte do contrato criado.

Reviewer: ````suggestion
- **Definidos abaixo**. Escolher quais meios de pagamento irão fazer parte do contrato.
````

4. Clique em "Salvar".
<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_meio_pagamento_contrato-7.gif)
<p>&nbsp;</p>

### Atribuindo meio de pagamento do contrato a um canal

Para **atribuir o meio de pagamento do contrato ao canal de sua loja**, siga os passos abaixo:

1. No painel administrativo, acesse "Canais" no menu Canais.

> 2. Localize o canal da sua loja e clique em "editar".

Reviewer: ````suggestion
2. Localize o canal da sua loja e clique em "Editar".
````

> ---

Reviewer: ````suggestion
````

>   indexable: false

Reviewer: ````suggestion
````

> ---

Reviewer: ````suggestion
````

---
 sites_supported:
  - mlb
---

# Linx Commerce

## O que é Linx Commerce?

> Linx Commerce é uma plataforma, que te permite receber pagamentos com o Mercado Pago em sua loja virtual.

Reviewer: ````suggestion
É uma **plataforma virtual que te permite receber pagamentos com o Mercado Pago**.
````

> Linx Commerce é uma plataforma de e-commerce, que te permite receber pagamentos através do Mercado Pago em sua loja virtual. 

Reviewer: ````suggestion
Linx Commerce é uma plataforma, que te permite receber pagamentos com o Mercado Pago em sua loja virtual. 
````

> Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_credito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) no próprio checkout da sua loja.

Reviewer: bookmark de cartão não está funcionando

> Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_credito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) no próprio checkout da sua loja.

Reviewer: ````suggestion
Com ela você poderá, oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_credito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) no próprio checkout da sua loja.
````

## Etapas para configurar

> Os **passos para começar a receber com Mercado Pago**, são os seguintes:

Reviewer: ````suggestion
Os passos para **começar a receber com Mercado Pago**, são os seguintes:
````

1. Crie uma [conta de vendedor](https://www.mercadopago[FAKER][URL][DOMAIN]/activities) no Mercado Pago caso ainda não tenha uma.
2. Adicione o Mercado Pago como meio de pagamento dentro da sua loja.

> 3. Configure a forma de pagamento Mercado Pago.

Reviewer: ````suggestion
3. Configure a forma de pagamento como Mercado Pago.
````

4. Configure o meio de pagamento do contrato.

## Adicione o Mercado Pago como meio de pagamento

Para **adicionar o Mercado Pago em sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Meios de pagamentos" no menu Configs.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.
````

2. Clique no botão “Adicionar meio de pagamento". 
3. Na lista, Fornecedor de Serviço, selecione o "Mercado Pago V2" e clique em "Próximo Passo".
4. Defina um **Nome** para o meio de pagamento Mercado Pago V2, selecione o status "Ativo".
<p>&nbsp;</p>

>     ![CORE Configuracao](/images/core/core_adicione_meio_pagamento-1.gif)

Reviewer: Aqui devemos mudar de CORE para Linx?

<p>&nbsp;</p>

## Configure a forma de pagamento

> Depois de adicionar o Mercado Pago, você terá a opção de oferecer pagamentos com [cartão de crédito](#bookmark_configurando_os_dados_de_cartão_de_credito) e [boleto bancário](#bookmark_configurando_os_dados_de_boleto) no próprio checkout de sua loja.

Reviewer: bookmark de cartão não está funcionando

> Além disso, você pode [oferecer parcelas sem juros](#bookmark_configurando_o_parcelamento_na_sua_conta_do_mercado_pago), consultar e configurar as [taxas e prazos](https://www.mercadopago[FAKER][URL][DOMAIN]/settings/release-options/) de suas vendas online quando quiser.

Reviewer: bookmark de parcelamento não está funcionando

### Configurando os dados da integração

Após adicionar o meio de pagamento, **configure os dados de sua integração com o Mercado Pago**, através dos seguintes passos:

> 1. Acesse a aba "Integração" e complete os campos **Public Key e Access Token Key** que correspondem às [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de produção de sua conta do Mercado Pago.

Reviewer: ````suggestion
1. Acesse a aba "Integração" e complete os campos **Public Key** e **Access Token Key** que correspondem às [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de produção de sua conta do Mercado Pago.
````

2. Preencha o nome que será informado na fatura de seu cliente.

> 3. Na seção **Modo teste**, selecione a opção "sim" caso queira habilitar transações de teste.

Reviewer: ````suggestion
3. Na seção **Modo teste**, selecione a opção "sim", caso queira habilitar transações de teste.
````

> 4. Complete os campos **Public Key e Access Token Key** de teste que correspondem às [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de Sandbox da sua conta do Mercado Pago.

Reviewer: ````suggestion
4. Complete os campos **Public Key** e **Access Token Key** de teste que correspondem às [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de Sandbox da sua conta do Mercado Pago.
````

5. Na seção de parcelamento, selecione "Externo (API do Mercado Pago)" onde você vai utilizar as condições de parcelamento diretamente da sua conta Mercado Pago.
6. Clique em "Salvar".
<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_configurando_integracao-2.gif)
<p>&nbsp;</p>

> > Encontre todas as informações sobre suas credenciais em [preguntas frecuentes](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/faqs/credentials/).

Reviewer: ````suggestion
> Encontre todas as informações sobre suas credenciais em [preguntas frequentes](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/faqs/credentials/).
````

### Configurando os dados de Cartão de Crédito

Para **configurar os dados de Cartão de Crédito em sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Meios de pagamentos" no menu Configs.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.
````

2. Localize o meio de pagamento criado e clique em "editar".

> 3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar e selecione o status "Ativo".

Reviewer: ````suggestion
3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar, e selecione o status "Ativo".
````

> 3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar e altere o status para **Ativo**.

Reviewer: ````suggestion
3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar e selecione o status "Ativo".
````

> 4. Preencha os campos correspondentes caso necessário e clique em "Salvar".

Reviewer: ````suggestion
4. Preencha os campos correspondentes, se necessário, e clique em "Salvar".
````

| Campos | Dados necessários |
|-------------------------------------------|---------------------------------------------------------------------------------------|

> | Porcentagem de juros | Caso deseje colocar uma porcentagem de juros, coloque um valor, caso não, deixe em 0,00. |

Reviewer: ````suggestion
| Porcentagem de juros | Caso queira adicionar uma porcentagem de juros, preencha um valor, caso não, deixe em 0,00. |
````

| Tipo de juros | Emissor - Incluir juros no total do pedido. Emissor - Juros acrescido pelo emissor. Lojista - Incluir juros no total do pedido. |
| Número total de parcelas | Coloque o número total de parcelas que será possível escolher por compra. |

> | Número de parcelas sem juros | O número total de parcelas sem juros que será possível escolher por compra. |

Reviewer: ````suggestion
| Número de parcelas **sem juros** | O número total de parcelas sem juros que será possível escolher por compra. |
````

> | Número de parcelas sem juros | O número total de parcelas sem juros que será possível realizar o pedido. |

Reviewer: ````suggestion
| Número de parcelas sem juros | O número total de parcelas sem juros que será possível escolher por compra. |
````

| Parcela mínima | Valor mínimo disponível para cada parcela de uma compra. |
| Identificador de Integração | Coloque um código que será identificado no seu ERP. |

> | Ordem | A ordem em que a bandeira aparecerá no Checkout. |

Reviewer: ````suggestion
| Ordem | A ordem em que as bandeiras aparecerão no Checkout. |
````

> | Caminho Imagem | O caminho de onde puxará a imagem para a bandeira do cartão. |

Reviewer: ````suggestion
| Caminho Imagem | O caminho onde as imagem das bandeiras do cartão serão buscadas. |
````

<br>

> WARNING
>
> Importante
>
> Ao utilizar configuração de parcelamento externo, todas as configurações de parcelamento realizadas na plataforma serão desconsideradas.

> > O parcelamento externo **não** será mostrado em listagens ou no detalhe do produto.

Reviewer: Aqui ficaria melhor por compra?
Ou melhor seguir por produto mesmo?

<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_cartao-3.gif)
<p>&nbsp;</p>

### Configurando os dados de Boleto

Para **configurar os dados de Boleto Bancário o em sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Meios de pagamentos" no menu Configs.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Meios de pagamentos" no menu Configs.
````

2. Localize o meio de pagamento criado e clique em "editar".

> 3. Acesse a aba "Boletos" e altere o status para **Ativo**.

Reviewer: ````suggestion
3. Acesse a aba "Boletos" e selecione o status "Ativo".
````

4. Preencha o campo **Identificador de Integração** com a informação que será identificada em seu ERP.
5. Informe a ordem em que o boleto aparecerá no seu checkout e clique em "Salvar".
<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_configurando_boleto-4.gif)
<p>&nbsp;</p>

> ## Configurando o parcelamento na sua conta do Mercado Pago.

Reviewer: ````suggestion
## Configurando o parcelamento na sua conta do Mercado Pago
````

1. Acesse a seção [Seu negócio](https://www.mercadopago.com.br/business/) na sua conta do Mercado Pago.
2. Procure por **Oferecer parcelamento sem juros** e clique em "Ativar".
3. Escolha quantas parcelas você quer oferecer e clique em “Ativar” para confirmar as alterações.
<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_parcelamento_conta-5.gif)
<p>&nbsp;</p>

> E pronto! Você está oferecendo parcelamento sem acréscimo, assumindo as tarifas de parcelamento que você configurou.

Reviewer: ````suggestion
E pronto! Você está oferecendo parcelamento sem juros, assumindo as tarifas de parcelamento que você configurou.
````

## Configurando meios de pagamento do contrato

O contrato tem a finalidade de determinar configurações específicas que serão utilizadas pela sua loja.

### Criando um contrato

Para **criar um contrato em sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Contratos" no menu Backoffice.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Contratos" no menu Backoffice.
````

2. Clique em “Adicionar Contrato".
3. Preencha um nome para o contrato que está sendo criado e clique em "salvar".
<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_criando_um_contrato-6.gif)
<p>&nbsp;</p>

### Definindo os meios de pagamento do contrato

Para **definir os meios de pagamento do contrato em sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Contratos" no menu Backoffice.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Contratos" no menu Backoffice.
````

2. Localize o contrato criado e clique em "editar".
3. Localize a sessão meios de pagamento e defina qual a opção que o contrato irá refletir para o meio de pagamento, sendo eles:

- **Todos os meios de pagamento ativos na plataforma.** Irá permitir que todos os meios de pagamento habilitados na plataforma sejam refletidos para este contrato.

> - **Definidos abaixo.** Escolher quais os meios de pagamento irão fazer parte do contrato criado.

Reviewer: ````suggestion
- **Definidos abaixo.** Escolher quais meios de pagamento irão fazer parte do contrato criado.
````

4. Clique em "Salvar".
<p>&nbsp;</p>
    ![LINX Configuracao](/images/linx/linx_meio_pagamento_contrato-7.gif)
<p>&nbsp;</p>

### Atribuindo meio de pagamento do contrato a um canal

Para **atribuir o meio de pagamento do contrato ao canal de sua loja**, siga os passos abaixo:

> 1. No painel administrativo de sua loja, acesse "Canais" no menu Canais.

Reviewer: ````suggestion
1. No painel administrativo, acesse "Canais" no menu Canais.
````

2. Localize o canal da sua loja e clique em "editar".

> 3. Na sessão contrato padrão, adicione o contrato criado anteriormente.

Reviewer: ````suggestion
3. Na sessão contrato padrão, adicione o contrato que criou.
````

4. Clique em "Salvar".
<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_atribuindo_contrato_ao_canal-8.gif)
<p>&nbsp;</p>

<!-- -->

> > Para mais informação, visite o [site oficial da Linx Commerce](https://docs.linxcommerce.com.br/docs).

Reviewer: ````suggestion
> Para mais informações, visite o [site oficial da Linx Commerce](https://docs.linxcommerce.com.br/docs).
````

### Configurando os dados da integração

Após ter adicionado o meio de pagamento, **continue configurando os dados de sua integração com o Mercado Pago em sua loja**, siga os seguintes passos:

1. Acesse a aba "Integração" e complete os campos "Public Key e Access Token Key" que correspondem as [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de "Produção" de sua conta "Mercado Pago" da seção "Integração".
2. Preencha o nome que será informado na fatura de seu cliente.
3. Na seção "Modo teste", selecione a opção "sim" caso queira habilitar transações de teste.
4. Complete os campos "Public Key de teste e Access Token Key de teste" que correspondem as [credenciais]([FAKER][CREDENTIALS][URL_BASIC]) de "Sandbox" de sua conta "Mercado Pago".
5. Na seção de "Parcelamento", selecione a configuração de parcelamento "Externo (API do Mercado Pago) onde utiliza as condições de parcelamento diretamente de sua conta Mercado Pago.
6. Clique em "Salvar".
<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_configurando_integracao-2.gif)
<p>&nbsp;</p>

### Configurando os dados de Cartão de Crédito

Para **configurar os dados de Cartão de Crédito em sua loja**, siga os seguintes passos:

> 1. No painel de administração de sua loja, acesse Meios de pagamentos no menu Configs.

Reviewer: ````suggestion
1. No painel administrativo de sua loja, acesse "Meios de pagamentos" no menu Configs.
````

> 2. Localize o meio de pagamento criado e clique em editar.

Reviewer: ````suggestion
2. Localize o meio de pagamento criado e clique em "editar".
````

> 3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar e altere o status para "Ativo".

Reviewer: ````suggestion
3. Acesse a aba "Cartões de Crédito", clique sobre a bandeira que deseja configurar e altere o status para **Ativo**.
````

4. Preencha os campos correspondentes caso necessário e clique em "Salvar".

| Campos | Dados necessários |
|-------------------------------------------|---------------------------------------------------------------------------------------|
| Porcentagem de juros | Caso deseje colocar uma porcentagem de juros, coloque um valor, caso não, deixe em 0,00. |
| Tipo de juros | Emissor - Incluir juros no total do pedido. Emissor - Juros acrescido pelo emissor. Lojista - Incluir juros no total do pedido. |
| Número total de parcelas | Coloque o número total de parcelas que será possível realizar o pedido. |
| Número de parcelas sem juros | O número total de parcelas sem juros que será possível realizar o pedido. |
| Parcela mínima | Valor mínimo disponível para cada parcela de uma compra. |
| Identificador de Integração | Coloque um código que será identificado no seu ERP. |
| Ordem | A ordem em que a bandeira aparecerá no Checkout. |
| Caminho Imagem | O caminho de onde puxará a imagem para a bandeira do cartão. |
<br>

> WARNING
>
> Importante
>
> Ao utilizar configuração de parcelamento externo, todas as configurações de parcelamento realizadas na plataforma serão desconsideradas.

> > O parcelamento externo NÃO será mostrado em listagens ou no detalhe do produto.

Reviewer: ````suggestion
> O parcelamento externo **não** será mostrado em listagens ou no detalhe do produto.
````

<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_configurando_cartao-3.gif)
<p>&nbsp;</p>

### Configurando os dados de Boleto

Para configurar os dados de Boleto Bancário o em sua loja, siga os seguintes passos:

> 1. No painel de administração de sua loja, acesse Meios de pagamentos no menu Configs.

Reviewer: ````suggestion
1. No painel administrativo de sua loja, acesse "Meios de pagamentos" no menu Configs.
````

> 2. Localize o meio de pagamento criado e clique em editar.

Reviewer: ````suggestion
2. Localize o meio de pagamento criado e clique em "editar".
````

> 3. Acesse a aba "Boletos" e altere o status para "Ativo".

Reviewer: ````suggestion
3. Acesse a aba "Boletos" e altere o status para **Ativo**.
````

> 4. Preencha o campo "Identificador de Integração" com  informação que será identificada em seu ERP.

Reviewer: ````suggestion
4. Preencha o campo **Identificador de Integração** com a informação que será identificada em seu ERP.
````

> 5. Informe a "Ordem" em que o boleto aparecerá em seu checkout e clique em "Salvar".

Reviewer: ````suggestion
5. Informe a ordem em que o boleto aparecerá no seu checkout e clique em "Salvar".
````

<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_configurando_boleto-4.gif)
<p>&nbsp;</p>

> ## Configurando o parcelamento em sua conta Mercado Pago.

Reviewer: ````suggestion
## Configurando o parcelamento na sua conta do Mercado Pago.
````

> 1. Acesse sua [conta Mercado Pago](https://www.mercadopago.com.br/business/) e clique em "Seu negócio"

Reviewer: ````suggestion
Acesse a seção [Seu negocio](https://www.mercadopago.com.br/business/) na sua conta do Mercado Pago.
````

> 2. Clique na opção “Configurações”, navegue até o campo "Oferecer parcelas sem acréscimo" e clique em “Ativar”.

Reviewer: ````suggestion
2. Procure por **Oferecer parcelamento sem juros** e clique em "Ativar".
````

> 3. Escolha “Quantas parcelas você quer oferecer?” e clique em “Ativar” para confirmar as alterações.

Reviewer: ````suggestion
3. Escolha quantas parcelas você quer oferecer e clique em “Ativar” para confirmar as alterações.
````

<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_parcelamento_conta-5.gif)
<p>&nbsp;</p>

E pronto! Você está oferecendo parcelamento sem acréscimo, assumindo as tarifas de parcelamento que você configurou.

## Configurando meios de pagamento do contrato

O contrato tem a finalidade de determinar configurações específicas que serão utilizadas pela sua loja.

### Criando um contrato

> Para **criar um contrato em sua loja**, siga os seguintes passos:

Reviewer: ````suggestion
Para **criar um contrato em sua loja**, siga os passos abaixo:
````

> 1. No painel de administração de sua loja, acesse Contratos no menu Backoffice.

Reviewer: ````suggestion
1. No painel administrativo de sua loja, acesse "Contratos" no menu Backoffice.
````

> 2. Clique no botão “Adicionar Contrato".

Reviewer: ````suggestion
2. Clique em “Adicionar Contrato".
````

> 3. Preencha um "Nome" para o contrato que está sendo criado e clique em salvar.

Reviewer: ````suggestion
3. Preencha um nome para o contrato que está sendo criado e clique em "salvar".
````

<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_criando_um_contrato-6.gif)
<p>&nbsp;</p>

### Definindo os meios de pagamento do contrato

Para **definir os meios de pagamento do contrato em sua loja**, siga os seguintes passos:

> 1. No painel de administração de sua loja, acesse Contratos no menu Backoffice.

Reviewer: ````suggestion
1. No painel administrativo de sua loja, acesse "Contratos" no menu Backoffice.
````

> 2. Localize o "Contrato" criado e clique em editar.

Reviewer: ````suggestion
2. Localize o contrato criado e clique em "editar".
````

> 3. Localize a sessão "Meios de Pagamento" e defina qual a opção que o contrato irá refletir para o meio de pagamento, sendo eles:

Reviewer: ````suggestion
3. Localize a sessão meios de Pagamento e defina qual a opção que o contrato irá refletir para o meio de pagamento, sendo eles:
````

- **Todos os meios de pagamento ativos na plataforma.** Irá permitir que todos os meios de pagamento habilitados na plataforma sejam refletidos para este contrato.
- **Definidos abaixo.** Escolher quais os meios de pagamento irão fazer parte do contrato criado.

4. Clique em "Salvar".
<p>&nbsp;</p>
    ![CORE Configuracao](/images/core/core_meio_pagamento_contrato-7.gif)
<p>&nbsp;</p>

### Atribuindo meio de pagamento do contrato a um canal

Para **atribuir o meio de pagamento do contrato ao canal de sua loja**, siga os seguintes passos:

> 1. No painel de administração de sua loja, acesse Canais no menu Canais.

Reviewer: ````suggestion
1. No painel administrativo de sua loja, acesse "Canais".
````

> 2. Localize o canal de sua loja e clique em editar.

Reviewer: ````suggestion
2. Localize o canal da sua loja e clique em "editar".
````

> 3. Na sessão "Contrato Padrão", adicione o contrato criado anteriormente.

Reviewer: ````suggestion
3. Na sessão contrato padrão, adicione o contrato criado anteriormente.
````
